sljit_src/sljitNativeARM_32.c

```diff
@@ -1005,7 +1005,7 @@ SLJIT_API_FUNC_ATTRIBUTE sljit_s32 sljit_has_cpu_feature(sljit_s32 feature_type)
 	case SLJIT_HAS_FPU:
 	case SLJIT_HAS_F64_AS_F32_PAIR:
 #ifdef SLJIT_IS_FPU_AVAILABLE
-		return SLJIT_IS_FPU_AVAILABLE;
+		return SLJIT_IS_FPU_AVAILABLE != 0;
```

Comment:
What is the aim of this change?

Reply:
`sljit_has_cpu_feature()` returns 1 if the feature is available, but `SLJIT_IS_FPU_AVAILABLE` can be set to any non zero value to mean "available".

before this change it was always assumed that `SLJIT_IS_FPU_AVAILABLE` will either be 0 or 1 which was not technically correct.

Reply:
Change to `(SLJIT_IS_FPU_AVAILABLE) != 0`, so it even can be an expression (some `syscall() & flag` for example).